daemon/cmd/daemon.go

```diff
@@ -475,12 +475,12 @@ func NewDaemon(ctx context.Context, cancel context.CancelFunc, epMgr *endpointma
 	}
 
 	identity.IterateReservedIdentities(func(_ identity.NumericIdentity, _ *identity.Identity) {
-		metrics.Identity.Inc()
+		metrics.Identity.WithLabelValues("reserved").Inc()
```

Comment:
Do you know if these metrics will double-count vs. the `AllocateIdentity()` `defer()`?

Reply:
I don't think so -- in my testing I did not see reserved identities being double counted (installed prometheus + grafana and plotted things). I'll try to look in the code to see the code path of `AllocateIdentity` if it gets called for reserved ones.

Reply:
Looked at the code and I was able to trace it to https://github.com/cilium/cilium/blob/master/pkg/identity/reserved.go#L49-L51. Given this I don't think they get allocated via the normal `AllocateIdentity()` method.